.github/workflows/docker_doublet-detection.yml

```diff
@@ -0,0 +1,63 @@
+name: Build Docker image for doublet-detection
+
+concurrency:
+  # only one run per branch at a time
+  group: "doublet-detection-${{ github.ref }}"
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - analyses/doublet-detection/Dockerfile
+      - analyses/doublet-detection/.dockerignore
+      - analyses/doublet-detection/renv.lock
+      - analyses/doublet-detection/conda-lock.yml
+  push:
+    tags:
+      - "v*.*.*"
+    branches:
+      - main
+    paths:
+      - analyses/doublet-detection/Dockerfile
+      - analyses/doublet-detection/.dockerignore
+      - analyses/doublet-detection/renv.lock
+      - analyses/doublet-detection/conda-lock.yml
+  workflow_dispatch:
+    inputs:
+      push-ecr:
+        description: "Push to AWS ECR"
+        type: boolean
+        required: false
+  workflow_call:
+    inputs:
+      push-ecr:
+        description: "Push to AWS ECR"
+        type: boolean
```

Comment:
This workflow is never called from other workflows, so this should not be needed.
```suggestion
```